### PR TITLE
Feat: health and resolve option

### PR DESCRIPTION
**Pull request by arcos**
Adds HEALTH and RESOLVE as seen in the Year Zero SRD.

fbl_health_resolve_setting

fbl_health_resolve_charsheet

### CODE REVIEW

**Comment by narukaioh** [comment]
So what's the motivation for this addition?

**Comment by narukaioh** [comment]
> So what's the motivation for this addition?

I believe the idea here in the system is to be strictly Forbidden Lands only

**Comment by aMediocreDad** [comment]
I think this is an OK thing to add. I am merging PRs that are related to fixes, along with changes I have made in order to ensure compatibility with new expansions coming out. 

I'll review this one once I have released a new version compatible with the modules. I will make extensive changes to the way characters work after the expansions are released and so, this will certainly be released together with said changes.

File: src/actor/actor-sheet.js [hunk 1]
```diff
@@ -13,6 +14,15 @@ export class ForbiddenLandsActorSheet extends ActorSheet {
 		data = this.computeItems(data);
 		data.carriedStates = this.#getCarriedStates();
 		data.gear = this.#filterGear(data.items);
+		if (!this.useHealthAndResolve) {
+			const filteredAttributes = Object.entries(data.system.attribute).filter(
+				([key, _]) => key !== "health" && key !== "resolve",
+			);
+			data.system.attribute = Object.fromEntries(filteredAttributes);
```
Comment (aMediocreDad) [inline src/actor/actor-sheet.js:21]: I think it is entirely OK that the "health" and "resolve" attributes exist in the data passed to the sheet even if they are not used. So all you need is the `data.system.useHealthAndResolve = <boolean>`

**Review by aMediocreDad — APPROVED** [review]
Like I mentioned I think this is fine. I'm happy to fold it into the system. I plan to move around on the setting and such before doing another release. 

Please consider the comments I have made above. At the very least remove the redundant handlebars helper before merging.

**Pull request merged**